release/models/system/openconfig-system.yang

```diff
@@ -908,6 +914,49 @@ module openconfig-system {
     }
   }
 
+  grouping system-macaddr-routing-config {
+    description
+      "Configuration data for system's routing MAC addresses.";
+
+    leaf routing-mac {
+      type oc-yang:mac-address;
+      description
+        "Any packets destined to this MAC address must be routed by the system,
+	in addition to any other MAC addresses that system may already have been
+	using for that purpose. Packets destined to this MAC address are expected
+	to go through ingress lookup and result in packet being routed by the
+	switch. It is not expected to be used as the source MAC address of the
```

Comment:
I think that we should probably say that they should be forwarded according with the result of that lookup -- if the action is to drop, then they should be dropped.